docs/references/general/fitacf.md

```diff
@@ -1,2 +1,175 @@
+| *ercod*                 | **None**         | ***short***  | Error Code                                                                              |
+| *stat.agc*              | **None**         | ***short***  | AGC status word                                                                         |
+| *stat.lopwr*            | **None**         | ***short***  | LOPWR status word                                                                       |
+| *noise.search*          | **None**         | ***float***  | Calculated noise from clear frequency search                                            |
+| *noise.mean*            | **None**         | ***float***  | Average noise from clear frequency band                                                 |
+| *channel*               | **None**         | ***short***  | Channel number for variant changes in radars parameters in one scan, ex) frequency      |
+| *bmnum*                 | **None**         | ***short***  | Beam number                                                                             |
+| *bmazm*                 | **None**         | ***float***  | Beam azimuth                                                                            |
+| *scan*                  | **None**         | ***short***  | Scan flag, switches between 0 to 1 or 0 to -1 to indicate a new scan                    |
+| *offset*                |  *us* | ***short***  | Offset between channels for a stereo radar                                                       |
+| *rxrise*                | *us* | ***short***  | Receiver rise time                                                                      |
+| *intt.sc*               | *s*        | ***short***  | whole number of seconds of integration time                                             |
+| *intt.us*               | *us* | ***int***    | Fractional number of microseconds of integration time                                    |
+| *txpl*                  | *us* | ***short***  | Transmitted pulse length                                                                |
+| *mpinc*                 | *us* | ***short***  | Multi-pulse increment                                                                   |
+| *mppul*                 | **None**         | ***short***  | Number of pulses in the sequence                                                        |
+| *mplgs*                 | **None**         | ***short***  | Number of lags in the sequence                                                          |
+| *mplgexes*              | **None**         | ***short***  |                                                                                         |
+| *ifmode*                | **None**         | ***short***  | Receiver was processing at an IF (intermediate frequency) instead of RF (radio frequency) band       |
```

Comment:
Is there a history on the description here? I would like to modify the text.

Reply:
I don't know what the history was, I just found it in DaVitpy comments. How did you want to modify it? 

I just hope we are not getting too pedantic about this and holding up the progress.